app/react/src/client/preview/types-6-0.ts

```diff
@@ -22,3 +29,16 @@ export type Meta<Args = DefaultArgs> = BaseMeta<ReactComponent> &
  */
 export type Story<Args = DefaultArgs> = BaseStory<Args, ReactReturnType> &
   Annotations<Args, ReactReturnType>;
+
+/**
+ * Object representing the preview.ts module
+ *
+ * Add something descriptive here
+ * @see [Unit testing with Storybook](https://storybook.js.org/docs/react/workflows/unit-testing)
+ */
+export type GlobalConfig = {
+  decorators?: any[]; // TODO: figure out a proper type here
```

Comment:
I need help here :D

Reply:
CC: @shilman do you know what decorators can return? is it just a react component or some other magic?

Reply:
https://github.com/storybookjs/storybook/blob/next/lib/addons/src/types.ts#L162-L164 ?